src/js/utils/MetronomeUtil.js

```diff
@@ -91,5 +91,36 @@ module.exports = {
     });
 
     return rootTree;
+  },
+
+  parseJob(job) {
+    let {history}= job;
+
+    if (history == null) {
+
```

Comment:
I only added those blank lines as I was told, that we should always add blank lines in those cases.  Now it turns out that this blank lines should only be there under some circumstances. This is a waste of time! Time we could've better use to fix bugs or ship features.
Let's please add rules where possible, and agree to not block any PR by stylistic comments which are not covered in .eslintrc rules. Either there is a rule, or the style is important enough to write our own eslint plugin.  In any other case, we should just stop complaining!